core/templates/dev/head/components/social_buttons/SocialButtonsDirective.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the social buttons displayed in footer
```

Comment:
Well, you forgot the 'the' in this comment ... but never mind. We can merge as-is.

Perhaps, in the future, you should double-check your code before submitting -- use tools like 'git diff' or 'meld' to ensure that the diff is what you want it to be, and make sure that everything is absolutely correct before you push. Also, after you've pushed, check the Files tab on GitHub to make sure everything is correct.

Note that you can also run the frontend checks locally. (In general, it's more efficient in the long run to do it properly, and a bit slower, than to do a rushed job that results in about 10 back-and-forths.)